src/lib/components/WellCompletions/components/Settings/FilterMenu.tsx

```diff
@@ -22,6 +22,7 @@ const FilterMenu: React.FC = React.memo(() => {
         <div>
             <Tooltip title={isDrawerOpen ? "Close filter menu" : "Filter"}>
                 <Button
+                    data-testid="filter_button"
```

Comment:
Avanti has also pointed out this new attribute can be removed using https://www.npmjs.com/package/babel-plugin-react-remove-properties. Maybe something to be refined in the future.